node/grpc/server_v2.go

```diff
@@ -198,13 +201,24 @@ func (s *ServerV2) StoreChunks(ctx context.Context, in *pb.StoreChunksRequest) (
 
 // validateStoreChunksRequest validates the StoreChunksRequest and returns deserialized batch in the request
 func (s *ServerV2) validateStoreChunksRequest(req *pb.StoreChunksRequest) (*corev2.Batch, error) {
+	if req.GetDisperserID() != api.EigenLabsDisperserID {
```

Comment:
Can we add a comment why this would result in an invalid store chunks request?

Reply:
This is better to have onchain information (i.e. what dispersers are registered), so dropped this, and leave it to authenticator.